Spring-Boot-News-Service/src/main/java/com/atomshermental/springbootnewsservice/NewsServiceImplementation.java

```diff
@@ -0,0 +1,69 @@
+package com.atomshermental.springbootnewsservice;
+
+import com.atomshermental.springbootnewsservice.Interfaces.NewsRepository;
+import com.atomshermental.springbootnewsservice.Interfaces.NewsService;
+import com.atomshermental.springbootnewsservice.model.*;
+import lombok.AllArgsConstructor;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.time.Instant;
+import java.util.*;
+
+@RequiredArgsConstructor
+@Service
+public class NewsServiceImplementation implements NewsService {
+
+    private final NewsRepository newsRepositoryImplementation;
+
+
+    public List<NewsShablon> getNewsList() {
+        List<News> list = newsRepositoryImplementation.getNewsList();
+        List<NewsShablon> newsShablon = list.stream().map(news -> new NewsShablon(news.getId(), news.getHeader())).toList();
+        return newsShablon;
+    }
+
+    public News getNews(Long id) {
+        return newsRepositoryImplementation.getNewsForId(id);
+    }
+
+    public List<Comment> getComments(Long id) {
+        return newsRepositoryImplementation.getNewsForId(id).getComments();
+    }
+
+    public void setComment(Long id, CommentRequest commentShablon) {
+        News news = newsRepositoryImplementation.getNewsForId(id);
+        List<Comment> comments = news.getComments();
+        Comment newComment = Comment.builder()
+                .author(commentShablon.getAuthor())
+                .date(Date.from(Instant.now()))
+                .text(commentShablon.getText())
+                .build();
```

Comment:
Create a factory with method for creating Comment object from CommentRequest object